js/acquia_lift.v2.js

```diff
@@ -7,7 +7,7 @@
 
 (function ($, Drupal) {
   var instance;
-  Drupal.acquiaLiftV2API = (function () {
+  Drupal.acquiaLiftAPI = (function () {
```

Comment:
I'm still trying to wrap my head around everything - but why is this still named ".v2.js"?  Seems like it should be replacing acquia_lift.api.js.